packages/docusaurus-plugin-openapi/src/docs/docs.ts

```diff
@@ -0,0 +1,223 @@
+/* ============================================================================
```

Comment:
I think we need to keep the original license headers. You may need to update eslint to allow different headers for these files. I think we also need to update this project’s license to include the original copy from Docusaurus’ license

Reply:
It may also be worth reaching out to Docusaurus and ask if they could expose all this

Reply:
@bourdakos1 It's a bit complicated because I had to remove all the logic that had to do with versions, which we don't support. So they would have to do some large changes to be able to export this in a way that is generic enough for us.

For now, I clarified that these files are taken and adapted from upstream. I also updated the license headers.